templates/default-distributed-db-config.j2

```diff
@@ -0,0 +1,11 @@
+{
+  "autoDeploy": {{orientdb_distributed_db_config.autoDeploy | to_json }},
+  "hotAlignment": {{orientdb_distributed_db_config.hotAlignment | to_json }},
+  "executionMode": "{{orientdb_distributed_db_config.executionMode}}",
+  "readQuorum": {{orientdb_distributed_db_config.readQuorum}},
+  "writeQuorum": {{orientdb_distributed_db_config.writeQuorum}},
+  "failureAvailableNodesLessQuorum": {{orientdb_distributed_db_config.failureAvailableNodesLessQuorum | to_json}},
+  "readYourWrites": {{orientdb_distributed_db_config.readYourWrites | to_json}},
+  "servers": {{orientdb_distributed_db_config.servers | to_json}},
```

Comment:
I do not see the "servers" variable at this level in documentation. I see it as part of cluster dictionary, is it a mistake or did I miss something ?